doozer

```diff
@@ -829,6 +889,50 @@ def images_scan_for_cves(runtime):
     subprocess.check_call(["atomic", "scan"] + image_urls)
 
 
+@cli.command("images:show-tree", short_help="Display the image relationship tree")
+@click.option(
+    "--imagename", default=False, is_flag=True,
+    help="Use the image name instead of the dist-git name")
+@click.option(
+    "--yml", default=False, is_flag=True,
+    help="Ouput to yaml formatted text, otherwise generate a tree view")
+@pass_runtime
+def images_print(runtime, imagename, yml):
```

Comment:
Yo dawg, I noticed this doesn't have a docstring, so it won't have a cool `--help` output. It's not _like super important or whatever_, but it's inconsistent.

Reply:
Will add. And that function is named wrong. Copy/Paste error... fixing.